edb/pypg_query/demo.txt

```diff
@@ -0,0 +1,100 @@
+$ python sql_to_pgast.py 'SELECT first_name, Person.* FROM Persons p JOIN Posts ON p.id = posts.posted_by WHERE LENGTH(posts.title) > 10'
```

Comment:
I suggest renaming to README.txt

Reply:
And adding other relevant info explaining what this subpackage does.

Reply:
Oh, I commited this file just to show sully how far this is.